recipes/cpr/all/conanfile.py

```diff
@@ -18,14 +17,36 @@ class CprConan(ConanFile):
         "shared": [True, False],
         "fPIC": [True, False],
         "with_openssl": [True, False],
+        "with_winssl": [True, False],
     }
     default_options = {
         "shared": False,
         "fPIC": True,
-        "with_openssl": True,
+        "with_openssl": False,
+        "with_winssl": True,
```

Comment:
Do you need to add `crypt32` to the system_libs in package_info?

Reply:
Should it not pick it up from curl and openssl?

Reply:
I guess it depends on the chosen libcurl backend:
https://github.com/conan-io/conan-center-index/blob/262b55fcc715000d980f0b58e6b1366df527348a/recipes/libcurl/all/conanfile.py#L561